picard/ui/mainwindow.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+﻿# -*- coding: utf-8 -*-
```

Comment:
Did your editor add a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark "byte order mark")? Or what changed on this line? BOMs are generally discouraged, and I don't see anything else that would have changed on this line.

Reply:
Good catch, indeed. I checked the file, there is a BOM. That should be removed for UTF-8 files.

Reply:
```suggestion
```